docs/guide.md

```diff
@@ -26,37 +26,18 @@ app.use(responseTime);
   while any code after is the "bubble" phase. This crude gif illustrates how async function allow us
   to properly utilize stack flow to implement request and response flows:
 
-![koa middleware](/docs/middleware.gif)
+![koa middleware](/docs/middleware.gif =100x)
```

Comment:
this doesn't work in GitHub markdown preview for me

Reply:
what's the `=100x` for?

Reply:
It's used on other editors, I was trying if it was working on github and forgot to remove it